RELEASES.md

```diff
@@ -297,6 +297,7 @@ Compiler
 - [Added the `tiny` value to the `code-model` codegen flag.][72397]
 - [Added tier 3 support\* for the `mipsel-sony-psp` target.][72062]
 - [Added tier 3 support for the `thumbv7a-uwp-windows-msvc` target.][72133]
+- [Upgraded to LLVM 10.][67759]
```

Comment:
You should add a link to the PR so that it's rendered correctly, like:
https://github.com/rust-lang/rust/blob/0d33ab7af4aebe786410b4c10367eb6ddf13af0b/RELEASES.md#L96-L98